test/lib/ansible_test/_data/sslcheck.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+"""Show openssl version."""
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import json
+
+
+def main():
+    """Main program entry point."""
+    # noinspection PyBroadException
+    try:
+        from ssl import OPENSSL_VERSION_INFO
+        version = list(OPENSSL_VERSION_INFO[:3])
+    except Exception:  # pylint: disable=broad-except
+        version = None
```

Comment:
nit: Does this need to be within `main()`?

nit: rather than catch `Exception` could it just be `TypeError` and maybe `IndexError`?

Reply:
Good catch on the import in main, I'll fix this. I added main after the other code and should have left the import outside.

As for the exception type, we don't know what all the possible failures are. I'd expect `ImportError` and `AttributeError`, but other errors could occur as well since `ssl` support isn't guaranteed. If the import or version lookup fails for any reason, we want to report `None` for the version. It's a best-effort version check.

Reply:
> As for the exception type, we don't know what all the possible failures are.

Makes sense. I always prefer to avoid catching `Exception` if possible, but it's not critical here.